core/templates/dev/head/pages/topic_viewer/TopicViewerNavbarBreadcrumbDirective.js

```diff
@@ -0,0 +1,31 @@
+// Copyright 2018 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Directive for the navbar breadcrumb of the topic viewer.
+ */
+oppia.directive('topicsViewerNavbarBreadcrumb', [
+  'UrlInterpolationService', function(UrlInterpolationService) {
+    return {
+      restrict: 'E',
+      scope: {
+        getTopicName: '@topicName',
```

Comment:
Did you mean '&topicName'?

Reply:
No, I'm printing topic_name that I'm getting from backend and using that to get using one-way binding.

Reply:
Try and find a way to not use Jinja for the topic name. Get it using AJAX instead. I think we do that in the collection editor, you may want to look at the code there.

Reply:
(In general try to avoid using Jinja, period.)

Reply:
Okay... It's just that I wanted to use one less API call. Done.